### ISSUE TITLE
how to stabilize training? 

### ISSUE THREAD

**Issue by fishman2008** [junyanz/pytorch-CycleGAN-and-pix2pix#354]
The L1 loss with pix2pix tends to fluctuate and not converging. The data has about 500 pairs of images. Other data augmentation schemes have been added. Any suggestions on how to reduce the loss for L1?

**Comment by junyanz** [comment]
You can probably increase the lambda for L1 loss if your main concern is about the convergence of L1 loss.

**Comment by fishman2008** [comment]
@junyanz thanks for the reply. I have tried with increasing the lambda, but with no observed benefits. The L1 loss still fluctuates and the synthesized results contain artifacts. I have used around 500 pairs of samples, comprising of 30 subjects. Do you think the number of subjects would be the problem? I have added some data augmentations in this case.

**Comment by junyanz** [comment]
The size of the dataset depends on your task. If your task is easy, a few hundreds of pairs are enough. But if your task is complicated, you may need more training data.  Also, the L1 loss may not be a good indicator of the quality.
